blog/2021-08-27-release-notes.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Luigi v1.16.0
+seoMetaDescription: Release notes for Luigi v1.16.0
+author:
+  - Aleksandra Simeonova
+layout: blog
+---
+
+You can read about the new features in Luigi v1.16.0 in the release notes below.
+
+<!-- Excerpt -->
+
+#### Extended intent based navigation
+
+A new API method [navigateToIntent](https://docs.luigi-project.io/docs/luigi-client-api/?section=navigatetointent) was introduced. The method internally generates a URL of the form `#?intent=<semantic object>-<action>?<param_name>=<param_value>` through the given input arguments. This then follows a call to the original `linkManager.navigate(...)` function. Consequently, the following calls will have the exact same effect:
+
+```javascript
+linkManager().navigateToIntent('Sales-settings', {project: 'pr2', user: 'john'})
+linkManager().navigate('/#?intent=Sales-settings?project=pr2&user=john')
+```
+
+#### newTab option in linkManager
+
+This is another improvement to the Luigi Client linkManager API. It is an option that allows you to open paths in a new window through Luigi Client linkManager:
+
+```javascript
+LuigiClient.linkManager().newTab().navigate('/projects/xy/foobar');
+```
+
+You can read more in the documentation [here](https://docs.luigi-project.io/docs/luigi-client-api/?section=newtab).
+
+
+#### addNavHrefs support for application switcher and product switcher
+
+You now have the possibility to apply [addNavHrefs](https://docs.luigi-project.io/docs/navigation-parameters-reference/?section=addnavhrefs) to the [app switcher](https://docs.luigi-project.io/docs/navigation-advanced?section=app-switcher) and [product switcher](https://docs.luigi-project.io/docs/navigation-advanced?section=product-switcher) in Luigi. This allows you to enable product switcher links to be opened in new tab, for example. For more information, see [PR #2172](https://github.com/SAP/luigi/pull/2172) and [PR #2177](https://github.com/SAP/luigi/pull/2177).
+
```

Comment:
I'd suggest to reemove it, since this is something we supposed to have long time ago when introduced first time addNavHrefs. It is more a bug fix then a new feature.